### PR TITLE
Collect issues

### PR DESCRIPTION
**Pull request by barbelity**

### CODE REVIEW

File: tasks/ArtifactoryCollectIssues/task.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    "minimumAgentVersion": "1.83.0",
+    "instanceNameFormat": "Artifactory Collect Build Issues",
+    "inputs": [
+        {
+            "name": "artifactoryService",
+            "type": "connectedService:artifactoryService",
+            "label": "Artifactory service",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Artifactory service used for fetching VCS information."
```
Comment (eyalbe4) [inline tasks/ArtifactoryCollectIssues/task.json]: Artifactory service used for fetching VCS information. -->
Artifactory service used for fetching information collected in previous builds.

File: tasks/ArtifactoryCollectIssues/task.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            "helpMarkDown": "To collect build issues, you need to provide a config file. \nTo read more about the issues-collection yaml schema click <a href=\"https://www.jfrog.com/confluence/display/JFROG/Artifactory+Azure+DevOps+Extension#ArtifactoryAzureDevOpsExtension-CollectBuildIssues\" target=\"_blank\">here</a>.",
+            "options": {
+                "taskConfiguration": "Task Configuration",
+                "file": "File"
+            }
+        },
+        {
+            "name": "taskConfig",
+            "type": "multiLine",
+            "label": "Config Yaml",
```
Comment (eyalbe4) [inline tasks/ArtifactoryCollectIssues/task.json]: "Config Yaml" --> "Issues collection configuration"

File: tasks/ArtifactoryCollectIssues/task.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            "visibleRule": "configSource=taskConfiguration",
+            "defaultValue": "",
+            "properties": {
+                "resizable": "true",
+                "rows": "10",
+                "maxLength": "10000"
+            },
+            "helpMarkDown": "Use config yaml from task configuration."
```
Comment (eyalbe4) [inline tasks/ArtifactoryCollectIssues/task.json]: Use config yaml from task configuration. -->
To specify the details required for the build issues collection, please create a yaml configuration as described <a href=\"https://www.jfrog.com/confluence/display/JFROG/Artifactory+Azure+DevOps+Extension\" target=\"_blank\">here</a>."

File: tasks/ArtifactoryCollectIssues/task.json [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        },
+        {
+            "name": "file",
+            "type": "filePath",
+            "label": "File path",
+            "visibleRule": "configSource=file",
+            "helpMarkDown": "Use config yaml from a local file."
```
Comment (eyalbe4) [inline tasks/ArtifactoryCollectIssues/task.json]: Use config yaml from a local file. -->
Path to a file, with the details required for the build issues collection. Please create a yaml configuration as described <a href=\"https://www.jfrog.com/confluence/display/JFROG/Artifactory+Azure+DevOps+Extension\" target=\"_blank\">here</a>."

File: tasks/ArtifactoryCollectIssues/collectIssues.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    // Get input parameters.
+    let artifactoryService = tl.getInput('artifactoryService', false);
+    let buildName = tl.getInput('buildName', true);
+    let buildNumber = tl.getInput('buildNumber', true);
+    let configSource = tl.getInput('configSource', false);
+
+    // Create config yaml.
+    let configPath = path.join(defaultWorkDir, 'issuesConfig' + Date.now() + '.yaml');
```
[the remark this answers is not in the document]
Comment (barbelity) [reply tasks/ArtifactoryCollectIssues/collectIssues.js]: Added.

**Comment by barbelity** [comment]
Fixed all input descriptions 👍

**Pull request merged**